llvm/docs/LangRef.rst

```diff
@@ -3030,8 +3030,10 @@ as follows:
 ``S<size>``
     Specifies the natural alignment of the stack in bits. Alignment
     promotion of stack variables is limited to the natural stack
-    alignment to avoid dynamic stack realignment. The stack alignment
-    must be a multiple of 8-bits. If omitted, the natural stack
+    alignment to avoid dynamic stack realignment.
+    The stack alignment must be must be in the range [1,2^16)
+    and must be a power of two times the byte width.
```

Comment:
I'm not very happy with this wording.
What I'm trying to say is that the value must be `ByteWidth * 2^N` ,without assuming that ByteWidth == 8.